docs/conf.py

```diff
@@ -325,12 +324,19 @@
     r"{ref}`bioinformatics <bioinformatics>`": r"bioinformatics",
     PROTO_REF_PATTERN: PROTO_REF_REPLACE,
     r"/protos/docs/service/index": r"/protos/docs/service/service",
+    r"<weather_forecasting>": r"</flytesnacks/weather_forecasting>",
```

Comment:
If `flytesnacks` adds another ref in `docs/tutorials.md`, we would still need to add a new entry in `flyte`. If this is true, can we document this somewhere?

An alternative is to parse `tutorials.md` and friends for anything that is `<SINGLE_WORD>` and replace it with `<flytesnacks/SINGLE_WORD>`.

Reply:
no we don't, the `weather_forecasting` page is really the only exception to this (tbh it should really be an example). In general all the docs should follow the `<auto_examples/...` pattern above.

Reply:
I see `weather_forecasting` and `environment_setup` having this pattern. For my understanding, if one adds a `<mlops_setup>` in `docs/tutorials.md` do we need to add the following to `REPLACE_PATTERNS`?

```python
r"<mlops_setup>": r"<flytesnacks/mlops_setup>"
```

Or we want this to be an anti-pattern and so this should not happen.

Reply:
Yeah this is an anti-pattern that we need to fix more systematically.

Eventually we can can specify the `list-table-toc` directive in these files and assume that all refs in these directives follow the pattern `<flytesnacks/examples/...>`. Any ref outside of `list-table-toc` won't be treated specially.

Basically any entries listed under `list-table-toc` in `tutorials`, `userguide`, and `integrations` ought to follow the example template described here: https://docs.flyte.org/projects/cookbook/en/latest/contribute.html